python_modules/libraries/dagster-dbt/dagster_dbt/cloud/resources.py

```diff
@@ -451,7 +451,13 @@ def run_job_and_poll(
         final_run_details = self.poll_run(
             run_id, poll_interval=poll_interval, poll_timeout=poll_timeout, href=href
         )
-        output = DbtCloudOutput(run_details=final_run_details, result=self.get_run_results(run_id))
+        try:
+            run_results = self.get_run_results(run_id)
+        # if you fail to get run_results for this job, just leave it empty
+        except Failure as _:
+            self._log.info("run_results.json not available for this run")
```

Comment:
```suggestion
            self._log.info("run_results.json not available for this run. Defaulting to empty result.")
```

Could we also fix the description in https://github.com/dagster-io/dagster/blob/2572d664b3c6c5389ca142f21b9b941d898d4d1c/python_modules/libraries/dagster-dbt/dagster_dbt/cloud/types.py#L21-L24? Seems like we don't use `None`, we use `{}`.